crates/rpc-trace-types/src/geth/mux.rs

```diff
@@ -0,0 +1,120 @@
+use crate::geth::{GethDebugBuiltInTracerType, GethDebugTracerConfig, GethTrace};
+use serde::{Deserialize, Serialize};
+use std::collections::HashMap;
+
+/// A `muxTracer` config that contains the configuration for running multiple tracers in one go.
+#[derive(Debug, Clone, Default, PartialEq, Eq, Serialize, Deserialize)]
+#[serde(rename_all = "camelCase")]
+pub struct MuxConfig {
+    #[serde(flatten)]
+    pub tracers: HashMap<GethDebugBuiltInTracerType, Option<GethDebugTracerConfig>>,
```

Comment:
can we also use tuple new type here like MuxFrame

Reply:
yea, good idea